project/ws/app/src/lib/head/left-menu/left-menu.component.scss

```diff
@@ -91,6 +92,11 @@
 //   margin: unset !important;
 // }
 
+::ng-deep  .mat-expansion-panel:not([class*=mat-elevation-z]) {
+    background: rgba(0, 0, 0, 0.02) !important;
```

Comment:
please add same on by creating a custom class as a parent

Reply:
Okay will do it